src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/Map/DataExtensions.cs

```diff
@@ -69,7 +74,8 @@ internal static ApprenticeshipIncentiveModel Map(this ApprenticeshipIncentive en
                 PausePayments = entity.PausePayments,
                 SubmittedDate = entity.SubmittedDate,
                 SubmittedByEmail = entity.SubmittedByEmail,
-                Status = entity.Status
+                Status = entity.Status,
+                MinimumAgreementVersion = new AgreementVersion(entity.StartDate)
```

Comment:
This will recalculate each time on retrieval and will be wrong if there has been a start date change of circumstance.  The constructor needs to be passed the int?.  

Add a static factory method to the value type to create the initial agreement e.g.

public static Agreement Create(startdate)
{
   // existing logic
}

then this would become

MinimumAgreementVersion = entity.MinimumAgreementVersion.HasValue ? new AgreementVersion(version) : AgreementVersion.Create(entity.StartDate);

Then there isn't any conflict with the CoC story.